src/main/java/net/floodlightcontroller/forwarding/Forwarding.java

```diff
@@ -607,6 +608,9 @@ protected Match createMatchFromPacket(IOFSwitch sw, OFPort inPort, OFPacketIn pi
                     if (FLOWMOD_DEFAULT_MATCH_TRANSPORT_DST) {
                         mb.setExact(MatchField.TCP_DST, tcp.getDestinationPort());
                     }
+                    if(FLOWMOD_DEFAULT_MATCH_TCP_FLAG){
```

Comment:
Could you please tell me how to judge switch type and ovs version in createMatchFromPacket()?